sample/CsmpAgentLib_sample.c

```diff
@@ -16,10 +16,7 @@
 
 #include <stdio.h>
 #include <stdbool.h>
-#include <stdlib.h>
```

Comment:
For any change being done to the sample App it is recommended to test the sample App build for other target platforms as well to ensure nothing is broken.